attributes/default.rb

```diff
@@ -23,3 +23,8 @@
 # Attributes for registering this service to consul
 default['kibana']['consul']['config_dir'] = '/opt/consul/etc'
 default['kibana']['consul']['bin'] = '/opt/bin/consul'
+default['consul']['cli_opts'] = {
+  'config-dir' => default['kibana']['consul']['config_dir'],
+  'enable-script-checks' => nil,
+  'advertise' => node['ipaddress']
```

Comment:
do we need this flags?

Reply:
do you mean `advertise` part ?
yes, we need this
because in existing test suite, the vagrant box have multiple private ip address
so the consul need to know which ip is used to be advertised to other node

Reply:
okay noted